FHICORC.Core/Services/Model/SmartHealthCardModel/Shc/SmartHealthCardImmunization.cs

```diff
@@ -20,12 +20,6 @@ public class SmartHealthCardImmunization
         public DateTime? OccurrenceDateTime { get; set; }
     }
 
-    public class CodeableConcept
```

Comment:
Didn't get it, why did you delete this one?

Reply:
Extended the class a bit and it was used by both Immunization and Observation. 
So I moved it into its own file 🙂

Reply:
Sure, thanks!